### DISCUSSION TITLE
Button Fixed Size

### DISCUSSION THREAD

**Discussion by FredericChang** [tailwindlabs/tailwindcss#8132] [Help] [answered]
Dear,

devDependencies=> "tailwindcss": "^3.0.24",

There're three rows in my attached picture. each row has two buttons.
Is it possible to fix the size of the button?
I've tried many ways, however, the size of the button will be varied according to the length of words.

![19](https://user-images.githubusercontent.com/7896940/163703116-4650be03-3097-4276-b841-da671cda622b.jpg)

`

                    <div class="grid grid-cols-6 gap-4 text-center">
                        <div class="col-start-2 col-span-4 font-sans text-3xl ">
                            <a>
                            COVID certificate for travel (for people living in this country)
                            </a>
                        </div>
                        <div class="col-start-2 col-end-6 col-span-2">
                            <a class="text-center">
                            A coronavirus pass allows you to travel safely to countries where a certificate is required. You can create a certificate if you’ve been vaccinated, if you’ve been tested and don’t have coronavirus, or if you’ve had coronavirus and have recovered.
                            </a>
                        </div>
                        <div class="col-start-1 col-end-7  col-span-8  ">
                            <button class=" text-sky-400/75 font-bold py-3 px-10  rounded-md border-1 border-indigo-500/100">Create A vaccination Certificate</button>
                            <button class="bg-indigo-500 hover:bg-indigo-700 text-white font-bold py-3 px-10 rounded-md place-self-center">I received a (booster) vaccination</button>
                        </div>
                        <div class="col-start-1 col-end-7  col-span-8">
                            <button class=" text-sky-400/75 font-bold py-3 px-10 rounded-md  h-15 w-200 border-1 border-indigo-500/100">Create A recovery Certificate</button>
                            <button class="bg-indigo-500 hover:bg-indigo-700 text-white font-bold py-3 px-10 rounded-md place-self-center">The test proves I have had coronavirus </button>                        </div>
                        <div class="col-start-1 col-end-7  col-span-8">
                            <button class=" text-sky-400/75 font-bold py-3 px-10 rounded-md border-1 border-indigo-500/100">Vaccination  appointment</button>
                            <button class="bg-indigo-500 hover:bg-indigo-700 text-white font-bold py-3 px-10 rounded-md place-self-center">Making an appointment</button>
                        </div>
                    </div>
`

**Comment by alexanderbluhm** [comment, marked as answer]
> the size of the button will be varied according to the length of words

This is because you use **padding** (`px-10`). To use a fixed width you can use Tailwinds `w-` utilities, e.g. `w-32`.
Here are the docs for more information: https://tailwindcss.com/docs/width

**Reply by FredericChang** [reply]
Thanks.
